### PR TITLE
Added mailbox instructions to profile

### PR DESCRIPTION
**Pull request by ahwnsals1234**
Added mailbox instructions to profile. One functional problem I found while coding was that because the tool tip contains too much information, the user cannot see the whole thing when the tab below a certain size. I see that there also can be more improvements made in styling. One of the idea that I had was, because tool tips are not accessible in the mobile version of Gordon 360, it might be a better idea for us to create a link instead of the tool tips.

Screenshot from 2023-06-02 10-14-45

### CODE REVIEW

File: src/components/BirthdayMessage/index.jsx
Comment (bennettforkner) [inline src/components/BirthdayMessage/index.jsx:36]: Please don't merge this change to the repository. I'm guessing you were just testing this feature.

File: src/views/Home/components/DiningBalance/index.jsx
Comment (bennettforkner) [inline src/views/Home/components/DiningBalance/index.jsx:175]: What is this change doing?

File: src/views/About/contributors.json
Comment (bennettforkner) [inline src/views/About/contributors.json:44]: This is unrelated to your changes. When this is added, it should include the entire team. I think you guys usually do it near the end of the program.

File: src/views/Timesheets/index.jsx
Comment (bennettforkner) [inline src/views/Timesheets/index.jsx:521]: This seems unrelated to the changes that you are making with mailboxes. It's best to try and stay within scope for one PR.

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (bennettforkner) [inline src/components/Profile/components/PersonalInfoList/index.jsx:43]: We prefer to delete code instead of commenting it out unless there is a good reason to save it. Since your pull request is open and ready for review, I would recommend deleting these lines.

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (bennettforkner) [inline src/components/Profile/components/PersonalInfoList/index.jsx:48]: Hard-coding colors into your code makes it hard to maintain in the long term. Check out `theme.ts` for the colors that are available to you.

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (bennettforkner) [inline src/components/Profile/components/PersonalInfoList/index.jsx:416]: These `fontSize` styles make for a lot of repeated code. Is there a better way to do this?

Comment (ahwnsals1234) [reply src/components/Profile/components/PersonalInfoList/index.jsx:416]: I was only focused on getting things to work yesterday, I will try to see if there is a more efficient way to do this. Thank you for the feedback.

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (ahwnsals1234) [reply src/components/Profile/components/PersonalInfoList/index.jsx:43]: Thank you for the feedback, I'll be more careful next time before submitting a PR.

File: src/components/BirthdayMessage/index.jsx
Comment (ahwnsals1234) [reply src/components/BirthdayMessage/index.jsx:36]: Thank you for all the feedback. I made a mistake of not creating a separate branch specifically for this project, so a lot of the changes in this PR were unrelated to the project. I will fix them right now.

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (ahwnsals1234) [reply src/components/Profile/components/PersonalInfoList/index.jsx:43]: Thank you for the feedback, I'll be more careful next time before submitting a PR.

File: src/components/BirthdayMessage/index.jsx
Comment (bennettforkner) [reply src/components/BirthdayMessage/index.jsx:36]: Never anything wrong with opening a PR for review! I'm just a little bit feisty on reviewing when I see a new PR :)

Comment (bennettforkner) [reply src/components/BirthdayMessage/index.jsx]: Never anything wrong with opening a PR for review! I'm just a little bit feisty on reviewing when I see a new PR :)

**Comment by ahwnsals1234** [comment]
@bennettforkner Made changes according to the given feedback. It'd be appreciated if you can take a look at the updates later when you have time please. Thank you.

**Review by bennettforkner — COMMENTED** [review]
Looks good to me, but I'll let one of your team members/leaders approve.

**Comment by bennettforkner** [comment]
Could you add a photo of what this feature looks like to the end-user on your description above? (You can edit the PR's description)

**Comment by ahwnsals1234** [comment]
Just added the photo to the PR, thanks Bennett!!

**Comment by ahwnsals1234** [comment]
Hey Bennett, sorry I just read this comment. I was following the same format for the other tooltip box which is the one above "Enter a Shift" note in Timesheets page. I can still fix the color to match the blue background color Gordon 360 mostly use if that would be preferred?

**Comment by bennettforkner** [comment]
> Hey Bennett, sorry I just read this comment. I was following the same format for the other tooltip box which is the one above "Enter a Shift" note in Timesheets page. I can still fix the color to match the blue background color Gordon 360 mostly use if that would be preferred?

No problem, no urgency for me.

I have actually never seen timesheets as a student and it's hard to use with the student.test account. Generally, I wouldn't use that codebase as an example as it is one of the lesser-maintained features that we have. I think it would be better as a GordonDialogBox though if that seems realistic to you. I welcome thoughts from other team members too!

**Comment by ahwnsals1234** [comment]
image
image

Updated the tooltip for the mailbox instructions to GordonDialogBox as mentioned in the feedback above.

**Comment by ahwnsals1234** [comment]
> image image
> 
> Updated the tooltip for the mailbox instructions to GordonDialogBox as mentioned in the feedback above.

This is just a thought, but I was wondering if we should use material icons instead of the instruction button that we currently have. I thought the icon might blend in better in that location but I'm assuming having the password view icon right next to it is something that we would also have to keep in mind.

**Review by russtuck — CHANGES_REQUESTED** [review]
This looks good to me.  It's helpful, and clean.  So, the only thing I think is required is to merge from develop and test to be sure that doesn't break anything.  

If you want to make another improvement, you can do that, but I'd gently suggest doing it in a separate pull request so this one can get in and get feedback from users.

**Comment by ahwnsals1234** [comment]
As a follow up, I made a different PR adding links to the video instructions, #1886

File: src/components/Profile/components/PersonalInfoList/index.jsx
Comment (russtuck) [inline src/components/Profile/components/PersonalInfoList/index.jsx:441]: Please keep all lines under 80 characters long.

**Review by russtuck — CHANGES_REQUESTED** [review]
One small change request, then please merge from develop so your branch is caught up.  Add a note that you've tested it after the merge and it still works, then I'll approve.

**Review by russtuck — APPROVED** [review]

**Pull request merged**
